lib/utils/database_helpers.rb

```diff
@@ -0,0 +1,66 @@
+# encoding: utf-8
+# author: Christoph Hartmann
+# author: Dominik Richter
+
+module DatabaseHelper
+  class SQLColumn
+    def initialize(row, name)
+      @row = row
+      @name = name
+    end
+
+    def value
+      @row[@name.downcase]
+    end
+
+    def to_s
+      'SQL Column'
+    end
+  end
+
+  class SQLRow
+    def initialize(query, row)
+      @query = query
+      @row = row
+    end
+
+    def column(column)
+      SQLColumn.new(@row, column)
+    end
+
+    def to_s
+      'SQL Row'
+    end
+  end
+
+  class SQLQueryResult
+    def initialize(cmd, results)
+      @cmd = cmd
+      @results = results
+    end
+
+    def empty?
+      @results.empty?
+    end
+
+    def stdout
```

Comment:
I'm nitpicking, but this and `stderr` really bug me because it feels like we're exposing implementation-level details to the user. They shouldn't care that we're making `command` calls underneath to get the data.

Can we call these `output` and `errors`? That way if we ever change the implementation, the method names will still be truthful.